docs/setup.md

```diff
@@ -1,4 +1,4 @@
-# Using Private Packagist in a Composer project
+# Composer Project Setup
```

Comment:
This change seems unrelated, and I kinda prefer the old one. Any particular reason for this change?

Reply:
@glaubinix Yeah sorry Nils and I also changed this because the link from within the navigation is also changed (no point in mentioning Private Packagist, we want to focus on Project Setup)